src/utils/PermissionsUtils.tsx

```diff
@@ -33,10 +33,27 @@ export const checkForAccessPermission = (
     return true;
   }
   if (showMessage) {
+    notification.error({
+      key: 'group-acl-warning',
+      message: `You are missing access to Group:ACL to read permissions`,
+      description: (
+        <p>
+          Go to{' '}
+          <a
+            href={`https://console.cognitedata.com/${sdk.project}/iam`}
+            target="_blank"
+            rel="noopener noreferrer"
+          >
+            Console
+          </a>{' '}
+          and set up any missing permissions or contact your administrator!
+        </p>
+      ),
+    });
     // eslint-disable-next-line no-console
     console.warn(`You are missing access to Group:ACL to read permissions`);
   }
-  return false;
+  return true;
```

Comment:
Basically we were blocking requests from going out, but it seems that sometimes users would have access to other things but not Groups (that should not happen...) so we need to enable this.

Reply:
Some people do have access to doing things without the groups access. It's also a possibility to only show an error if they try to do something they don't have access to?

Reply:
We do display errors when things go wrong, I have a proposal on how to standardize this across discovery during the architecture discussion though!